drivers/ethernet/phy/phy_mii.c

```diff
@@ -226,11 +227,24 @@ static int update_link_state(const struct device *dev)
 		if (phy_mii_reg_read(dev, MII_BMSR, &bmsr_reg) < 0) {
 			return -EIO;
 		}
-	} while (!(bmsr_reg & MII_BMSR_AUTONEG_COMPLETE));
+
+	if (!(bmsr_reg & MII_BMSR_AUTONEG_COMPLETE)) {
+		if (data->autoneg_timeout-- == 0U) {
+			LOG_DBG("PHY (%d) auto-negotiate timedout",
+				    cfg->phy_addr);
```

Comment:
Nit:

```suggestion
			LOG_DBG("PHY (%d) auto-negotiate timedout", cfg->phy_addr);
```

Reply:
@pdgendt is it safe to use the "Commit suggestion" button or should I apply the change manually, amending the original commit?

Reply:
It needs to be amended, so I think option 2 is easiest.